### DISCUSSION TITLE
How uninstall this or upgrade to Git 2.26 ?

### DISCUSSION THREAD

**Discussion by paulgureghian** [community/community#23327] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/19c57cf050ee36ee2c88d44f525ec054ffd6b642.png" data-download-href="/uploads/short-url/3FZ3hv0sxih4M74Ua2F5VXxJyIa.png?dl=1" title="git.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/8146561/181076150-8a36eb26-dbcf-427c-9145-daeb796d67af.png" alt="git.PNG" data-base62-sha1="3FZ3hv0sxih4M74Ua2F5VXxJyIa" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/19c57cf050ee36ee2c88d44f525ec054ffd6b642_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/1X/19c57cf050ee36ee2c88d44f525ec054ffd6b642_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/19c57cf050ee36ee2c88d44f525ec054ffd6b642.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/19c57cf050ee36ee2c88d44f525ec054ffd6b642_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">git.PNG</span><span class="informations">1280×800 19.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by guntrip** [comment, marked as answer]
<p>Hi there Paul,</p>
<p>It looks you have <a href="https://github.com/msysgit/msysgit" rel="nofollow noopener">msysgit</a> installed which has been superseded by Git For Windows:</p>
<p><a href="https://gitforwindows.org/" rel="nofollow noopener">https://gitforwindows.org/</a></p>
<p>If you download and install this, this will upgrade Git to version 2.26.1 on your machine. </p>
